crates/ruff_db/src/vendored.rs

```diff
@@ -79,6 +80,16 @@ impl VendoredFileSystem {
     }
 }
 
+impl fmt::Display for VendoredFileSystem {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let inner_locked = self.inner.lock();
+        let inner_borrowed = inner_locked.borrow();
+        f.debug_struct("VendoredFileSystem")
+            .field("data", &inner_borrowed)
+            .finish()
+    }
+}
```

Comment:
It's unclear to me what the benefit of implementing `Display` is. `Display` is mainly used as a representation shown to the user and it's unclear to me where this is needed. And if we need it, I think `debug_struct` isn't the right representation for it.

Reply:
Fair enough, I removed `Display`!